### PR TITLE
fix:password reset css changed issue fixed

### PR DESCRIPTION
**Pull request by binaryarun**

### CODE REVIEW

**Review by marizoldi — APPROVED** [review]

File: frontend/styles/partials/_passwordreset.scss [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 .passwordreset {
   background: #F0F2F4;
```
Comment (yanabar) [inline frontend/styles/partials/_passwordreset.scss]: can you please check if #F0F2F4 exists in _variables.scss? If so please use the variable
please do the same for other colours in your scss

**Comment by yanabar** [comment]
@marizoldi we had to make couple of more changes but @armsarun could do that in another branch shall we deploy to heroku to see if everything looks good ?

**Comment by marizoldi** [comment]
yes, there should also be the inline validation but I thought to merge this as I need to check an issue with migrations that I get.

**Comment by marizoldi** [comment]
looks fine on heroku - just pushed. Can you check too?

**Comment by yanabar** [comment]
Looks good yes. Only back to erehwon link and inline validation dont work. We can do this as a diff fix

**Pull request merged**